admin/app/views/solidus_admin/base/unauthorized.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1><%= t('solidus_admin.errors.authorization.access_denied.title') %></h1>
+<p><%= t('solidus_admin.errors.authorization.access_denied.description') %></p>
```

Comment:
Thoughts on adding some TW style to this so it fits in with the context?

Reply:
![Screenshot 2023-10-06 at 17 47 57](https://github.com/solidusio/solidus/assets/19948291/b4988f5d-dce8-4dfa-a80f-6e91b2eae1cf)